onadata/apps/main/views.py

```diff
@@ -1445,3 +1446,29 @@ def get_context_data(self, **kwargs):
         context['user'] = self.request.user
         context['request_path'] = self.request.get_full_path()
         return context
+
+
+class CustomLoginView(LoginView):
+    """
+    Custom LoginView subclass
+    """
+
+    def get_template_names(self, **kwargs):
+        """
+        get custom login template name if use_custom_login_template is true
+        """
+        if settings.USE_CUSTOM_LOGIN_TEMPLATE:
+            return settings.CUSTOM_LOGIN_TEMPLATE_PAGE
```

Comment:
This method needs to return a list see: https://docs.djangoproject.com/en/2.1/ref/class-based-views/mixins-simple/#django.views.generic.base.TemplateResponseMixin.get_template_names